recipes/coincurve/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "coincurve" %}
+{% set version = "19.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MementoRC/coincurve/archive/refs/tags/pre-v{{ version }}.tar.gz
+  sha256: 6d6045f0b6662d7f16c319ba7637206a5c3c6673e1e32d5a2f3dfc2b37888405
```

Comment:
This recipe needs to use a tagged release from ofek/coincurve before it is merged.

Reply:
Updated